### DISCUSSION TITLE
黑暗模式，启动加载logo显示不出来，是灰色的

### DISCUSSION THREAD

**Discussion by Dashan-IZ** [ChatGPTNextWeb/NextChat#854] [Q&A]
![image](https://user-images.githubusercontent.com/78125220/232349201-c4ec949f-4a32-43da-8c02-317ce651b534.png)
黑暗模式，启动加载logo显示不出来，是灰色的，希望作者能改改

**Comment by Yidadaa** [comment]
已知样式问题，但并不影响使用

**Reply by Dashan-IZ** [reply]
是的，但是看着怪难受的
